pkg/server/config.go

```diff
@@ -18,7 +18,15 @@ type (
 		GRPCPort string `mapstructure:"grpc_port"`
 		HTTPPort string `mapstructure:"http_port"`
 
-		Cors Cors `mapstructure:"cors"`
+		Timeouts Timeouts `mapstructure:"timeouts"`
+		Cors     Cors     `mapstructure:"cors"`
+	}
+
+	// Timeouts represents collection of different timeout regarding net/http.Server.
+	Timeouts struct {
+		WriteTimeoutSecond int `mapstructure:"write_timeout_second"`
```

Comment:
Instead of adding time precision here in the config, probably is good idea to have a configuration as `time.Duration` data type. Then we could pass configuration parameters like `"1s"`, `"1m"`, `"100ms"` etc. Viper will handle that for us. [Example](https://github.com/scribd/go-sdk/blob/422da470e754cd8e23a37bf6730d4a9bcbdeb07d/pkg/pubsub/config.go#L25)

Reply:
done